### PR TITLE
Move `BlockHelper.cs` from nekoyume unity to lib9c

### PR DESCRIPTION
**Pull request by riemannulus**
The title says it all

### CODE REVIEW

**Comment by limebell** [comment]
Build seems fails 😢

**Comment by riemannulus** [comment]
@limebell I fix build fail.

File: Lib9c/BlockHelper.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            GoldDistribution[] goldDistributions)
+        {
+            if (!tableSheets.TryGetValue(nameof(GameConfigSheet), out var csv))
+            {
+                throw new KeyNotFoundException(nameof(GameConfigSheet));
+            }
+            var gameConfigState = new GameConfigState(csv);
+            var redeemCodeListSheet = new RedeemCodeListSheet();
+            redeemCodeListSheet.Set(tableSheets[nameof(RedeemCodeListSheet)]);
+
```
Comment (moreal) [inline Lib9c/BlockHelper.cs:33]:
```suggestion
```

File: Lib9c/BlockHelper.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        /// 중 <see cref="GameAction.Id"/>를 제외하고 계산한 <see cref="HashDigest{T}"/>.</returns>
+        private static HashDigest<SHA256> GetHashOfFirstAction(Block<PolymorphicAction<ActionBase>> block)
+        {
+            var initializeStatesAction = (InitializeStates)block.Transactions.First().Actions[0].InnerAction;
+            Bencodex.Types.Dictionary plainValue = (Bencodex.Types.Dictionary) initializeStatesAction.PlainValue;
+            plainValue = (Bencodex.Types.Dictionary) plainValue.Remove((Text)"id");  // except GameAction.Id.
+            var bytes = plainValue.EncodeIntoChunks().SelectMany(b => b).ToArray();
+            return Hashcash.Hash(bytes);
+        }
+    }
```
Comment (moreal) [inline Lib9c/BlockHelper.cs:89]: In my thought, it seems not required for lib9c.

File: Lib9c/BlockHelper.cs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            if (!tableSheets.TryGetValue(nameof(GameConfigSheet), out var csv))
+            {
+                throw new KeyNotFoundException(nameof(GameConfigSheet));
+            }
+            var gameConfigState = new GameConfigState(csv);
+            var redeemCodeListSheet = new RedeemCodeListSheet();
+            redeemCodeListSheet.Set(tableSheets[nameof(RedeemCodeListSheet)]);
+
+
+            // FIXME 메인넷때는 따로 지정해야합니다.
```
Comment (moreal) [inline Lib9c/BlockHelper.cs:35]: If this comment is useful, then how about translating to English?

Comment (riemannulus) [reply Lib9c/BlockHelper.cs]: Fix at e217c27

File: Lib9c/BlockHelper.cs (hunk 1 above)
Comment (riemannulus) [reply Lib9c/BlockHelper.cs:27]: Fix at e217c27

**Review by moreal — APPROVED** [review]

**Review by longfin — APPROVED** [review]

**Pull request merged**